libraries/botframework-connector/src/auth/skillValidation.ts

```diff
@@ -221,4 +230,20 @@ export namespace SkillValidation {
         // whitelist AuthenticationConfiguration
         // We may need to add a ClaimsIdentityValidator delegate or class that allows the dev to inject a custom validator.
     }
+
+    /**
+     * Creates a set of claims that represent an anonymous skill. Useful for testing bots locally in the emulator
+     */
+    export function createAnonymousSkillClaim(): ClaimsIdentity {
+        // return new ClaimsIdentity(new List<Claim> { new Claim(AuthenticationConstants.AppIdClaim, AuthenticationConstants.AnonymousSkillAppId) }, AuthenticationConstants.AnonymousAuthType);
+        return new ClaimsIdentity(
+            [
+                {
+                    type: AuthenticationConstants.AppIdClaim,
+                    value: AuthenticationConstants.AnonymousSkillAppId,
+                },
+            ],
+            true // TODO auth type?
```

Comment:
Same story - should this be true (`isAuthenticated`)? This is where the C# code uses `AuthenticationConstants. AnonymousAuthType`, but we don't track the auth type in the constructor. Is that stored as a JWT claim?